domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterModule.kt

```diff
@@ -25,4 +28,13 @@ class PlatformParameterModule {
     return platformParameterSingleton.getBooleanPlatformParameter(SPLASH_SCREEN_WELCOME_MSG)
       ?: PlatformParameterValue.createDefaultParameter(SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE)
   }
+
+  @Provides
+  @MyDownloads
+  fun provideMYDownloadsParam(
+    platformParameterSingleton: PlatformParameterSingleton
+  ): PlatformParameterValue<Boolean> {
+    return platformParameterSingleton.getBooleanPlatformParameter(MY_DOWNLOADS_FLAG)
+      ?: PlatformParameterValue.createDefaultParameter(MY_DOWNLOADS_IS_ENABLED_DEFAULT)
```

Comment:
@ARJUPTA should this just be creating the default parameter since we have no intention to control this flag (yet) with Oppia's backend?

Reply:
Yes. Any flag which is not defined in the backend will be using default value always. But keeping the provider method in this format gives us the flexibility to introduce that flag in the backend at any point of time